cayenne-server/src/main/java/org/apache/cayenne/query/SQLSelect.java

```diff
@@ -263,6 +263,7 @@ protected Query createReplacementQuery(EntityResolver resolver) {
 		template.setDefaultTemplate(getSql());
 		template.setCacheGroup(cacheGroup);
 		template.setCacheStrategy(cacheStrategy);
+		template.addPrefetch(prefetches);
```

Comment:
This cause `NullPointerException` when no prefetches are set. This is visible in test also.